README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
Algorithm Tuning (parameter tuning?) Started looking at this. Ran the sample code, and it works. Looked a little bit into the estimator interface and exactly what GridSearchCV does. Came across this link: http://scikit-learn.org/stable/tutorial/machine_learning_map/
Helpful guide to pick an estimator. 

Next step is to try different parameters on the ridge function to see what I get.